File: _challenges/6696c2ca763f0712c198dbf8_challenge.js
// NAMEEN:The unpredictability of the game
// NAMERU:Непредсказуемость игры

/*
INSTRUCTIONENSTART

INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-8.png)

Турист в Лас-Вегасе начал игру со какой-то суммой денег.

Создайте переменную `startMoney` и присвойте ей значение большее либо равное 100.

В конце игры он проиграл какую-то сумму денег.

Создайте переменную `lostMoney` и присвойте ей значение от 50 до 80.

Создайте переменную `endMoney` и присвойте ей вычисленное значение оставшихся денег, как разницу между `startMoney` и `lostMoney`.

Похожий пример:
```javascript
const start = 55;
const lost = 10;
const end = start - lost;
```
INSTRUCTIONRUEND
*/

/*
HINTENSTART
```javascript
const startMoney = 100;
const lostMoney = 76;
const endMoney = startMoney - lostMoney;
```
HINTENEND
*/

/*
HINTRUSTART
```javascript
const startMoney = 100;
const lostMoney = 76;
const endMoney = startMoney - lostMoney;
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART
const
INITIALSOLUTIONEND
*/

// SOLUTIONSTART
const startMoney = 100;
const lostMoney = 76;
const endMoney = startMoney - lostMoney;
// SOLUTIONEND

// OPENTESTSSTART
it('The `startMoney` variable contains a value more or equal 100', () => {
  expect(startMoney).gte(100);
});

it('The `lostMoney` variable contains a value between 50 and 80', () => {
  expect(lostMoney).gte(50).lte(80);
});

it('The `endMoney` variable contains the difference between `startMoney` and `lostMoney`', () => {
  expect(endMoney).equal(startMoney - lostMoney);
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND

File: _challenges/6696c2ca763f0712c198dbf8_challenge.py
// NAMEEN:Double name
// NAMERU:Двойное имя

/*
INSTRUCTIONENSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-9.png)

Often Americans have two personal names.

Suppose the variable `name` stores the first name `Emma`.

The variable `name` needs to get the first two names separated by a space.

Write a command that changes the existing value of the `name` variable by concatenating it with the name 'Kate' with a space.
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-9.png)

Часто американцы имеют два личных имени.

Предположим, что в переменной `name` уже хранится первое имя `Emma`.

Требуется в переменной `name` получить два первых имени через пробел.

Запишите команду, которая изменяет имеющееся значение переменной `name`, соединяя его с именем 'Kate' через пробел.
INSTRUCTIONRUEND
*/

/*
HINTENSTART
Similar example:
```python
name = "Alice"
name = name + " " + "Marie"
```
HINTENEND
*/

/*
HINTRUSTART
Похожий пример:
```python
name = "Alice"
name = name + " " + "Marie"
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART
name = "Emma"
INITIALSOLUTIONEND
*/

// SOLUTIONSTART
name = "Emma"
name = name + " " + "Kate"
// SOLUTIONEND

// OPENTESTSSTART
it('The "name" variable contains a value Emma Kate', () => {
    expect(name).eq('Emma Kate');
});

it('The "name" is type of string', () => {
    expect(typeof name).eq('string');
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND
